src/types/BeforeSwapDeltas.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Could you do the rename first and then the changes to the name inside the file after? its hard to analyze the diff when it shows it as an entirely new file